<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Digital Recipe Book</string>
  </property>
  <centralwidget>
   <widget class="QWidget" name="centralWidget">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableView" name="recipesTableView"/>
     </item>
     <item>
      <widget class="QLineEdit" name="nameLineEdit">
       <property name="placeholderText">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="categoryLineEdit">
       <property name="placeholderText">
        <string>Category</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="ingredientsTextEdit">
       <property name="placeholderText">
        <string>Ingredients</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="instructionsTextEdit">
       <property name="placeholderText">
        <string>Instructions</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addRecipeButton">
       <property name="text">
        <string>Add Recipe</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchLineEdit">
       <property name="placeholderText">
        <string>Search by name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </centralwidget>
 </widget>
 <resources/>
 <connections/>
</ui>
